File: 791A-BearAndBigBrother/791a-bearAndBigBrother.c
#include <stdio.h>
 
int main(){
	int limak, bob, i = 0;
	scanf("%d %d", &limak, &bob);
	while(1){
		limak = 3*limak;
		bob = 2*bob;
		i++;
		if(limak > bob){
			printf("%d",i);
			break;
		}
	}
}

File: 791A-BearAndBigBrother/791a-bearAndBigBrother.py
if __name__ == "__main__":
  weights = input().split()
  limak = int(weights[0])
  bob = int(weights[1])

  i = 0
  while(True):
      limak = 3*limak
      bob = 2*bob
      i += 1
      if limak > bob:
          print(i)
          break
